yjit/src/stats.rs

```diff
@@ -787,3 +774,9 @@ fn global_allocation_size() -> usize {
     let stats = GLOBAL_ALLOCATOR.stats();
     stats.bytes_allocated.saturating_sub(stats.bytes_deallocated)
 }
+
+// Increase vm_insns_count. vm_exec_core uses this.
+#[no_mangle]
+pub extern "C" fn rb_yjit_add_vm_insns_count(count: u64) {
+    incr_counter_by!(vm_insns_count, count);
+}
```

Comment:
We could also get rid of this function if the insn count variable is owned/managed by the C side?

Reply:
removed this too.